Assets/Scripts/UIManager.cs

```diff
@@ -69,4 +86,46 @@ public void LevelText(int currentLevel)
     {
         _levelText.text = "Level: " + currentLevel.ToString();
     }
+
+    public void OptionsMenuButton()
+    {
+        if (_isOptionsMenuOpen == false)
+        {
+            OpenOptionsMenu();
+        }
+        else if (_isOptionsMenuOpen == true)
```

Comment:
```suggestion
        else if (_isOptionsMenuOpen)
```

shorthand for checking boolean conditions. You don't need to explicitly write out the `true` check here, it's redundant.